### PR TITLE
Add forms

### PR DESCRIPTION
**Pull request by Oklukeok**
### **_Features:_**
Added new functionalities:
-Create new comments
-Create new likes

image

image

image

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -4,6 +4,29 @@ def index
...
+    @user = User.find(params[:user_id])
     @post = Post.find(params[:id])
   end
+
+  def new
+    @post = Post.new(id: current_user.id)
+  end
+
+  def create
+    @post = Post.new(post_params)
```
Comment (afanimartin) [inline app/controllers/posts_controller.rb]: - New Post should be created by a logged in user. Please update the code to
```ruby
@post = @current_user.posts.new(post_params)
```

File: app/controllers/comments_controller.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
+class CommentsController < ApplicationController
+  def new
+    @comment = Comment.new
+  end
+
+  def create
+    @comment = Comment.new(comment_params)
```
Comment (afanimartin) [inline app/controllers/comments_controller.rb]: - Please only allow logged in users to create comments
```ruby
@comment = @current_user.comments.new(params)
```

**Review by afanimartin — CHANGES_REQUESTED** [review]
Hi @Oklukeok, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 🎉 🎉 🎉 
- Codebase is professional and organised
- All required routes are added

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

### Martin Afani
#### Slack: whiz25

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/controllers/comments_controller.rb (hunk 2 above)
Comment (Oklukeok) [reply app/controllers/comments_controller.rb]: @afanimartin Thanks for the hints! have a nice weekend!

**Review by fobadara — APPROVED** [review]
Hi @Oklukeok, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✅ linters are passing

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**